### DISCUSSION TITLE
Unhandled Exception: Build files not found when creating MSIX package for Flutter Windows app on Codemagic

### DISCUSSION THREAD

**Discussion by ZiyadgodilDev** [codemagic-ci-cd/codemagic-docs#2526] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Description

I am encountering an issue while creating an MSIX package for my Flutter Windows application using Codemagic. The error occurs during the `flutter pub run msix:create` command, which complains about missing build files.

#### Error Details

When running the following command:
```sh
flutter.BAT pub run msix:create --build-windows false --version 1.0.36.0 --publisher CN=DD7E57A6-EB9F-4526-AA2D-FA14583E51DD --publisher-display-name Zeymo --identity-name Zeymo.ZeymoPOS --store
```

### Expected results
```
> flutter.BAT pub run msix:create --build-windows false --install-certificate false
building msix files...                  ?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?10.0s
packing msix files...                   ?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?\?|?/?-?4.6s
msix created: build\windows\x64\runner\Release\zeymo_pos.msix
```

### Actual results
```
Unhandled exception:
Build files not found at C:\Users\builder\clone\build\windows\x64\runner\Release, first run "flutter build windows" then try again
#0      Configuration.validateWindowsBuildFiles (package:msix/src/configuration.dart:365:7)
<asynchronous suspension>
#1      Msix._buildMsixFiles (package:msix/msix.dart:111:5)
<asynchronous suspension>
#2      Msix._createMsix (package:msix/msix.dart:102:5)
<asynchronous suspension>
#3      Msix.create (package:msix/msix.dart:59:5)
<asynchronous suspension>
#4      main (file:///C:/Users/builder/AppData/Local/Pub/Cache/hosted/pub.dev/msix-3.16.7/bin/create.dart:4:3)
<asynchronous suspension>
```

### Build id (optional)

66435cf32c0be085bf32f264

**Comment by dtrdic** [comment]
hey @ZiyadgodilDev based on your latest builds, it seems that you have managed to publish your `.msix` file successfully. So I'll close this thread as resolved, if that's not the case please reopen the thread. Have a nice day

EDIT: By the way, if you need any help with the issue, please [book a call](https://calendly.com/david-fcu/30min) with me so we can find the solution together
